### DISCUSSION TITLE
Can't login, getting http proxy error

### DISCUSSION THREAD

**Discussion by willymmerlet12** [danny-avila/LibreChat#343] [Troubleshooting] [answered]
When I start the project, I get this error when trying to log in to my account : 
[vite] http proxy error at /api/endpoints:
Error: connect ECONNREFUSED ::1:3080
at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1494:16)

I use an external proxy and MongoDB with my keys filled for bing, chatgpt and openai.

**Comment by danny-avila** [comment]
Hey, thanks for checking out the project! I need a bit more information to help.

Can you share your api/.env file and client/.env files (omitting credentials)? How are you trying to log in? How are you starting the app?

I was actively making changes today, so pull from latest as of now, then follow the regular setup instructions from one of the guides. Docker is the easiest and most reliable.

**Comment by danny-avila** [comment]
Sorry for the inconvenience, I just merged another hotfix: #322 

I merged a lot of PRs today including dependency updates.

**Comment by willymmerlet12** [comment, marked as answer]
Awesome, thank you for the quick response. I will try again with the new changes and I'll let you know
